Dockerfile.devkit

```diff
@@ -0,0 +1,31 @@
+FROM debian:buster
+
+ENV LANG=C.UTF-8
+
+RUN set -x \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        docker.io \
+        gcc \
+        git \
+        jq \
+        libcurl4-openssl-dev \
+        libssl-dev \
+        python3-dev \
+        python3-pip \
+        python3-setuptools
+
+ADD requirements-dev.txt /marathon-lb/requirements-dev.txt
```

Comment:
Yes, since this is the image that runs the tests, it needs to have the python environment setup. It is added separately, since the codedir is added after the `pip3` install. This helps caching since in development it is expected for the code to change frequently, and reinstalling the python dependancies takes about 1 min. By adding the requirements early and installing them before adding the code, code changes only trigger a rebuild of the upper image (copying the code into an image) which takes a second or two vs the longer pause for `pip3` installation.